web/src/message_store.ts

```diff
@@ -121,9 +121,11 @@ export type Message = (
     sent_by_me: boolean;
     reply_to: string;
 
-    // These properties are used in `message_list_view.js`.
-    message_reactions: MessageCleanReaction[];
-    url: string;
+    // These properties are set and used in `message_list_view.js`.
+    // TODO: It would be nice if we could not store these on the message
+    // object and only reference them within `message_list_view`.
```

Comment:
It might be better for them to be just set when initializing message objects or mutating reactions/linkifiers.